Added chinese and spanish language

siyunesweek opened the pull request:
I think it is all, sorry it took me more time to learn how git works cause i got some problems.

Kr0oked reviewed (changes requested):
Thanks a lot for creating the MR!

There is one thing that might not be correct, the folder names of the chinese translation. I think it should be `app/src/main/res/values-zh` and `fastlane/metadata/android/zh-CN` .
Could you change this detail in a follow up commit? Or alternatively we can merge it as it is and I correct it afterwards.

Oh and I think the file app/src/main/res/values-v29/values-cn-zh/strings.xml is not needed.

siyunesweek commented:
I made a new commit 
  -Changed <code>app/src/main/res/values-cn-zh</code> to  <code>app/src/main/res/values-zh-rCN</code>
  -Deleted <code>app/src/main/res/values-v29/values-cn-zh</code> 
Sorry my fault.

Kr0oked commented:
Sorry for bringing this up again but the directory names are still not correct.

For spanish the values directory should be `values-es-rES` with an `r` in it. The current metadata directory `es-ES` for spanish is correct.

And I am a little bit unsure what would be correct for your chinese translation.

Unfortunately the android resource directory and the metadata directory use different formats and there are multiple Chinese language styles. I guess your translation is for 'Chinese Simplified' and not 'Chinese Traditional' or 'Chinese Hongkong', is this assumption correct?

For the metadata directory we can use `zh-CN`, `zh-TW` or `zh-HK` all without an `r` in it.
I think for the metadata directory `zh-CN` is the correct one, then the store entry is for 'Chinese Simplified'.

For the resource directory it works a little bit differently. I see that a lot of projects use `values-zh` then it is available for all regions. But we could use `values-zh-rCN` to restrict it for the China region.
![grafik](https://github.com/Kr0oked/Metronome/assets/5984532/dfb96283-dfd6-433f-bc36-1c0ec038662e)

Sorry for making it complicated ...

siyunesweek commented:
Don't worry, thanks to you for helping me.
- I changed <code>values-es-ES</code> to <code>values-es-rES</code>. 
- It is Simplified Chinese so I change it in the metadata <code>zh-rCN</code> to <code>zh-CN</code>
- For the resource directory I changed for now to <code>values-zh</code>. If in the future someone wants to add one for Singapore or others you can changed it after.

Hope this time goes well.
